contracts/ERC721CM.sol

```diff
@@ -588,11 +594,12 @@ contract ERC721CM is IERC721M, ERC721ACQueryable, Ownable, ReentrancyGuard {
     }
 
     /**
-     * @dev Returns data hash for the given minter, qty and timestamp.
+     * @dev Returns data hash for the given minter, qty, waiveMintFee and timestamp.
      */
     function getCosignDigest(
         address minter,
         uint32 qty,
+        bool waiveMintFee,
```

Comment:
do we have ongoing mints that will break the cosigner server?

Reply:
`getCosignDigest` is only invoked inside contract - we could have mark it internal actually. `getCosignNonce` is the only method called by our FE and it keeps the unchanged for 721 / newly added for 1155. 

Re cosigning service, the interface changed and `waiveMintFee` is optional:
```
export interface ICosignRequest {
  collectionContract: string;
  minter: string;
  qty: number;
  chainId: number;
  nonce: number;
  waiveMintFee?: boolean;
}
```

So it should be safe for ongoing mints.